jetty-util/src/main/java/org/eclipse/jetty/util/Scanner.java

```diff
@@ -216,7 +216,7 @@ else if (_filter == null || _filter.accept(f.getParentFile(), f.getName()))
 
             if (accepted)
             {
-                scanInfoMap.put(f.getCanonicalPath(), new MetaData(f.lastModified(), f.isDirectory() ? 0 : f.length()));
+                scanInfoMap.put(f.getAbsolutePath(), new MetaData(f.lastModified(), f.isDirectory() ? 0 : f.length()));
```

Comment:
Line 206 above is testing for f.getCanonicalPath() - you need to check the Scanner for all of these occurrences if changing to f.getAbsolutePath().